frontend/src/actions/TableActions.js

```diff
@@ -511,8 +520,12 @@ function updateCollapsedRows({ tableId, rows }) {
       collapsedRows,
       collapsedParentRows,
     } = table;
-    let newCollapsedParentRows = [];
-    let newCollapsedRows = [];
+
```

Comment:
I would add a short comment on how this is actually working, like we've had for filters.

Reply:
I already detailed this in the params